src/components/MatchProfile/MatchProfileForm.js

```diff
@@ -2,7 +2,25 @@ import Grid from '@mui/material/Grid';
 import Stack from '@mui/material/Stack';
 import AllChipsArray from "./AllChipsArray";
 import UserTypeButtons from "./UserTypeButtons";
+import { Button, ButtonGroup } from '@mui/material';
 
+const add_user_band_func = (user_id) => {
```

Comment:
use camelCase in JS

Reply:
What is camelCase?

Reply:
Nevermind I changed it. It's like no spaces and underscore and capitalize every first letter of the words. See it now.

Reply:
almost. You used pascal case. The first letter of the first word should be lower case